libraries/c_sdk/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -164,8 +233,20 @@ static bool _mqttSubscription_setUnsubscribe( const IotLink_t * pSubscriptionLin
 
     return true;
 }
+/*Generate Id for packet*/
```

Comment:
please add a line and separator(/*-----------------------------------------------------------*/) between 2 functions